mne/source_estimate.py

```diff
@@ -1036,8 +1016,8 @@ def __repr__(self):  # noqa: D105
         s += ", tmin : %s (ms)" % (1e3 * self.tmin)
         s += ", tmax : %s (ms)" % (1e3 * self.times[-1])
         s += ", tstep : %s (ms)" % (1e3 * self.tstep)
-        s += ", data size : %s x %s" % self.shape
-        return "<SourceEstimate  |  %s>" % s
+        s += ", data size : %s" % ' x '.join([str(x) for x in self.shape])
```

Comment:
maybe we should just say `data shape %s' % (self.shape,)`, it's more numpy/pythonic but still readable?